_docs/reference/release-notes.md

```diff
@@ -1,13 +1,84 @@
 ---
 title: Release Notes
-overview: What's been happening with Istio.
+overview: What's been happening with Istio
 
 order: 50
 
 layout: docs
 type: markdown
 ---
 
+## Istio 0.2
+
+#### General
+
+- **Updated Config Model**. Istio now uses the Kubernetes Custom Resource Definition model to describe and store its configuration.
+When running in Kubernetes environment, configuration can now be optionally managed using the `kubectl` command.
+
+- **Multiple Namespace Support**. Istio control plane components are now in the dedicated "istio-system" namespace. Istio can manage 
+services in other non-system namespaces.
+
+- **Mesh Expansion**. Initial support for adding non-Kubernetes services (in the form of VMs and/or physical machines) to a mesh. This is an early version of
+this features which has limitations (such as requiring a flat network across containers and VMs).
+
+- **Multi-Platform Support**. Initial support for running Istio outside Kubernetes using Consul or Eureka as service registries. This feature can handle 
+services across multiple service registries.
+
+#### Perf and Quality
+
+There have been many performance and reliability improvements throughout the system. We don’t consider Istio 0.2 ready for production yet, but we’ve made excellent progress in that direction. Here are a few items of note:
+
+- **Caching Client**. The Mixer client library used by Envoy now provides caching for Check calls and batching for Report calls, considerably reducing 
+end-to-end overhead.
+
+- **Avoid Hot Restarts**. The need to hot-restart Envoy has been mostly eliminated through effective use of LDS/RDS/CDS/EDS.
+
+- **Reduced Memory Use**. Significantly reduced the size of the sidecar helper agent, from 50Mb to 7Mb.
+
+- **Improved Mixer Latency**. Mixer now clearly delineates configuration-time vs. request-time computations, which avoids doing extra setup work at 
+request-time for initial requests and thus delivers a smoother average latency. Better resource caching also contributes to better end-to-end perf.
+
+- **Reduced Latency for Egress Traffic**. We now forward traffic to external services directly from the sidecar.
+
+#### Traffic Management
+
+- **Egress Rules**. It’s now possible to specify routing rules for egress traffic.
+
+- **New Protocols**. Mesh-wide support for WebSocket connections, MongoDB proxying.
+
+- **Other Improvements**. Ingress properly supports gRPC services, interfacing headless services, better support for health checks, and
+Jaeger tracing.
+
+#### Policy Enforcement & Telemetry
+
+- **Ingress Policies**. Policies can now be applied to north-south traffic in addition to east-west traffic supported in 0.1.
```

Comment:
Slightly awkward sentence, "in addition to east-west traffic supported in 0.1"

Maybe "in addition to east-west traffic polices that we supported in 0.1"? Something like that?

Reply:
Or maybe follow the same model you did below for HTTP vs. TCP?